docs/user/metric-parameters.md

```diff
@@ -1,6 +1,6 @@
 # Metric parameters
 
-All metric types must include the following required parameters:
+## Required metric parameters
 
 - `type`: **Required.**  Specifies the type of a metric, like "counter" or "event". This defines which operations are valid for the metric, how it is stored and how data analysis tooling displays it. See the list of [supported metric types](metrics/index.md).
```

Comment:
Putting "Required" next to each one... good for emphasis, or needless duplication?